src/main/java/org/opentripplanner/apis/gtfs/GraphQLScalars.java

```diff
@@ -10,18 +10,24 @@
 import graphql.schema.CoercingParseValueException;
 import graphql.schema.CoercingSerializeException;
 import graphql.schema.GraphQLScalarType;
+import java.text.ParseException;
+import java.time.OffsetDateTime;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import javax.annotation.Nonnull;
 import org.locationtech.jts.geom.Geometry;
 import org.opentripplanner.framework.geometry.GeometryUtils;
 import org.opentripplanner.framework.graphql.scalar.DurationScalarFactory;
 import org.opentripplanner.framework.model.Grams;
+import org.opentripplanner.framework.time.OffsetDateTimeParser;
 
 public class GraphQLScalars {
 
   private static final ObjectMapper geoJsonMapper = new ObjectMapper()
     .registerModule(new JtsModule(GeometryUtils.getGeometryFactory()));
   public static GraphQLScalarType durationScalar = DurationScalarFactory.createDurationScalar();
 
-  public static GraphQLScalarType polylineScalar = GraphQLScalarType
+  public static final GraphQLScalarType polylineScalar = GraphQLScalarType
```

Comment:
Use uppercase for constants: `POLYLINE_SCALAR`

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/5660/commits/964a3f767d6196524126102fbd46006427a3a7e5